client/components/docs-example/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+
+const DocsExample = ( {
```

Comment:
Why did you decide to use a function here, instead of a class with few short methods like `toggleButton` and `count`?